### ISSUE TITLE
Managing Sets

### ISSUE THREAD

**Issue by sarahjeansweeney** [NEU-Libraries/cerberus#1061]
It looks like the repository staff group is being given edit permissions for every new Set. This is useful for managing Sets, but the repository staff group now has more than 30 collaborative sets:

![2016-11-18_1010](https://cloud.githubusercontent.com/assets/4663073/20434720/37145c1e-ad77-11e6-96f0-81fc5cc89a55.png)

This Set, for example, was created by a regular user: https://repository.library.northeastern.edu/sets/neu:cj82np33j
The rights record for the set includes the repository staff group:

```
<rightsMetadata xmlns="http://hydra-collab.stanford.edu/schemas/rightsMetadata/v1" version="0.1">
<copyright>
<human type="title"/>
<human type="description"/>
<machine type="uri"/>
</copyright>
<access type="discover">
<human/>
<machine/>
</access>
<access type="read">
<human/>
<machine/>
</access>
<access type="edit">
<human/>
<machine>
<group>northeastern:drs:repository:staff</group>
<person>[nuid]</person>
</machine>
</access>
<embargo>
<human/>
<machine/>
</embargo>
</rightsMetadata>
```

This creates a lot of unnecessary noise in repository staff's collaborative sets list, and it can take more than 40 seconds for the Sets modal to appear after clicking "Save for Later" for any file (for me, anyway, it may be less for others).

Not everyone on the DRS staff group needs to manage collaborative sets, so I think:

- the repository staff group should be removed from sets where it wasn't explicitly selected (I can identify these)
- the ability to manage all sets should be moved to an admin feature. 

Similar to "Manage Communities", the "Manage Sets" admin page could include a list of all Sets with options to edit or delete.

**Comment by elizoller** [comment]
This all sounds reasonable to me but obviously David will need to weigh in.

Regarding the time for the Sets modal to appear, I think that might have more to do with how many groups a user belongs to as opposed to how many sets are actually available to the user. Maybe David has a better way of structuring the query which basically does a solr query where the edit_access_group is each of the users groups ORed together.

**Comment by dgcliff** [comment]
Sounds fine to me - @elizoller I'll look at the query to see if we can speed it up for @sarahjeansweeney

**Comment by dgcliff** [comment]
@sarahjeansweeney if you can get the list of sets where :staff was explicitly chosen, I'll remove the group from the others, and alter the code for future sets

**Comment by sarahjeansweeney** [comment]
Of all the sets I can see, I think this is the only one that should have the staff group: 
https://repository.library.northeastern.edu/sets/neu:cj82kf90c

**Comment by dgcliff** [comment]
Cool, that's easy enough then

**Comment by dgcliff** [comment]
Should be fixed with da4a17432db258ec5bd5da72c0ce02947aef09b0 and 5f4540814be910f5ea2dbd80d325ce3e53e573f9
